### DISCUSSION TITLE
JWT expiry limit

### DISCUSSION THREAD

**Discussion by ptrtorain** [supabase/supabase#7926] [Questions] [answered]
Can you remove the session length limit? I don't want my users to have to log in to my store. This will reduce site traffic and sales

**Comment by silentworks** [comment, marked as answer]
You can increase the JWT expiry limit from your [Dashboard](https://app.supabase.com/project/_/auth/settings). If you want this to be removed completely this is not possible as this is not how JWTs work.

**Reply by silentworks** [reply]
This is how JWTs work, its for security reasons.

Also can you keep your replies under this answer as it is related to this answer please.

**Comment by ptrtorain** [comment]
I know, but one week is not enough. It would be nice if the session lasted at least three months
